src/load/azext_load/tests/latest/test_load_test.py

```diff
@@ -335,6 +339,128 @@ def test_load_test_create(self, rg, load, vnet):
         except Exception as e:
             assert "InvalidNetworkConfigurationException" in str(e)
 
+    @ResourceGroupPreparer(**rg_params)
+    @LoadTestResourcePreparer(**load_params)
+    @VirtualNetworkPreparer(**vnet_params)
+    @StorageAccountPreparer(**sa_params)
+    def test_load_test_autostop(self, rg, load, vnet):
+        self.kwargs.update(
+            {
+                "test_id": LoadTestConstants.CREATE_TEST_ID,
+                "load_test_config_file": LoadTestConstants.LOAD_TEST_CONFIG_FILE,
+                "description": LoadTestConstants.DESCRIPTION,
+                "display_name": LoadTestConstants.DISPLAY_NAME,
+                "engine_instance": LoadTestConstants.ENGINE_INSTANCE,
+            }
+        )
+        checks = [
+            JMESPathCheck("testId", self.kwargs["test_id"]),
+            JMESPathCheck(
+                "loadTestConfiguration.engineInstances", self.kwargs["engine_instance"]
+            ),
+            JMESPathCheck("description", self.kwargs["description"]),
+            JMESPathCheck("displayName", self.kwargs["display_name"]),
+            JMESPathCheck("autoStopCriteria.autoStopDisabled", True),
+        ]
+        # Create load test with autostop disabled thru config file
```

Comment:
Can we have some more tests with config file.. enable  setting time window/ error rate etc?

Also can we add test where we are we have values coming from config as well as cmd.. we should ensure that the value is being picked from cmd in case of conflicts.

Reply:
Valid test cases
1. autostop disabled thru config file
2. autostop criteria thru command line arguments
3. autostop disabled thru command line arguments
4. autostop criteria thru config file
5. autostop criteria thru config file: only error rate provided
6. autostop criteria thru config file: only time window provided
7. autostop criteria when both config file and CLI arguments are provided
8. config file has autostop criteria and CLI argument is --autostop disable